vhdbuilder/packer/install-dependencies.sh

```diff
@@ -458,19 +468,25 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.7-hotfix.20200326
 1.15.10-hotfix.20200408.1
 1.15.11-hotfix.20200529.1
+1.15.12-hotfix.20200623.1
 1.16.7-hotfix.20200601.1
 1.16.8.1
 1.16.9-hotfix.20200529.1
+1.16.10-hotfix.20200623.1
 1.17.3-hotfix.20200601.1
 1.17.4.1
 1.17.5.1
+1.17.7-hotfix.20200624
 1.18.1.1
 1.18.2.1
+1.18.2-hotfix.20200624
+1.18.4-hotfix.20200624
 "
 for KUBERNETES_VERSION in ${PATCHED_HYPERKUBE_IMAGES}; do
   CONTAINER_IMAGE="mcr.microsoft.com/oss/kubernetes/hyperkube:v${KUBERNETES_VERSION}"
   pullContainerImage "docker" ${CONTAINER_IMAGE}
   echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
+  docker run --rm --entrypoint "" ${CONTAINER_IMAGE}  /bin/bash -c "iptables --version" | grep -v nf_tables && echo "Hyperkube contains no nf_tables" || (echo "Hyperkube contains nf_tables" && exit 99)
```

Comment:
exit on nf_tables as it's not working with ubuntu

Reply:
what if "docker run" fails?

Reply:
the build will also fail